examples/fizzbuzz.peggy

```diff
@@ -33,12 +33,15 @@ line
   / buzz
 
 fizzbuzz = f:fizz _ b:buzz { return f + b }
-fizz = @"fizz"i !{ return currentNumber % 3 }
-buzz = @"buzz"i !{ return currentNumber % 5 }
+\u0066izz 'fizz' = @"fizz"i !{ return currentNumber % 3 }
```

Comment:
Oh god not another language of this.

![image](https://user-images.githubusercontent.com/2983969/173265535-86855895-9dcb-4431-80b9-1e72d5cfe904.png)

Reply:
Honestly I'd rather _ban_ everything outside of `[a-z_]i[a-z0-9_]i`.